networkx/algorithms/tests/test_walks.py

```diff
@@ -0,0 +1,64 @@
+"""Unit tests for the :mod:`networkx.algorithms.walks` module."""
+
+import pytest
+
+import networkx as nx
+
+pytest.importorskip("numpy")
+pytest.importorskip("scipy")
+
+
+class TestNumberOfWalks:
+    @classmethod
+    def setup_class(cls):
```

Comment:
I'd also advocate for getting rid of the test class here, especially since we're down to a single function. I've always found OO tests harder to reason about and more fraught with corner cases.

In this case it should be as simple as:
 1. Removing the class
 2. Removing the `self` arguments from the test methods so that they become test functions instead
 3. Replacing the `setup_class` with either a) a pytest fixture that creates the graph or b) simply creating the graph in each test case (since it's really not that expensive in this case).

This isn't a blocker though - the tests could go in with the current organization (once the other comments have been addressed) and we can re-organize them in a follow-up PR!